timesketch/frontend-ng/src/components/Explore/AggregateDialog.vue

```diff
@@ -0,0 +1,622 @@
+            </v-col>
+            <v-col align="center">
+              <v-card outlined height="480px" :loading="!statsReady">
+                <v-card-title>
+                  Rare &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp; values
+                  <v-spacer></v-spacer>
+                  <v-tooltip bottom>
+                    <template v-slot:activator="{ on }">
+                      <v-icon color="primary" v-on="on">mdi-information-outline</v-icon>
+                    </template>
+                    <span>Rare &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp;
+                      events that have a maximum event count of 5
+                    </span>
+                  </v-tooltip>
+                </v-card-title>
+                <v-card-text>
+                  <v-data-table
+                    :headers="termHeaders"
+                    :items="rareValues"
+                    :items-per-page="10"
+                    :hide-default-footer="(rareValues.length <= 10)"
+                    dense
+                  >
+                  </v-data-table>
+                </v-card-text>
+              </v-card>
+            </v-col>
+        </v-row>
+        <v-row>
+          <v-col align="center">
+            <v-card outlined height="480px" :loading="!statsReady">
+              <v-card-title>
+                Percentage of &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp; events
+                <v-spacer></v-spacer>
+                <v-tooltip bottom>
+                  <template v-slot:activator="{ on }">
+                    <v-icon color="primary" v-on="on">mdi-information-outline</v-icon>
+                  </template>
+                  <span>Shows the ratio of &nbsp;<span style="font-family: monospace">{{ eventValue }}</span>&nbsp;
+                    events to other &nbsp;<span style="font-family: monospace">{{ eventKey }}</span>&nbsp; events.
+                  </span>
+                </v-tooltip>
+              </v-card-title>
+              <v-card-text v-if="statsReady">
+                <apexchart
+                  height="350px"
+                  :options="this.donutChartOptions"
+                  :series="this.donutChartSeries"
+                ></apexchart>
+              </v-card-text>
+            </v-card>
+          </v-col>
+
+          <v-col align="center">
+            <v-card outlined height="480px" :loading="!statsReady">
+              <v-card-title>
+                Event distribution by {{ this.distributionIntervals[this.selectedDistributionIntervalIndex] }}
+                <v-spacer></v-spacer>
+                <v-tooltip bottom>
+                  <template v-slot:activator="{ on }">
+                    <v-icon color="primary" v-on="on">mdi-information-outline</v-icon>
+                  </template>
+                  <span>Shows the distribution of &nbsp;<span style="font-family: monospace">{{ eventValue }}</span>&nbsp; 
```

Comment:
```suggestion
                  <span>Shows the distribution of <span style="font-family: monospace">{{ this.truncateValue(eventValue) }}</span>
```
* Remove the `&nbsp;` and adds the `truncateValue()` function.